dashboard/app/views/levels/editors/fields/_dsl_fields.html.haml

```diff
@@ -48,6 +48,40 @@
       The Lab2 implementation can host non-Lab2 sublevels, but only transitions
       between Lab2 levels will be done without a page reload.
 
+  - if @level.is_a?(Multi)
+    %h4 Tip: Allow Multiple Attempts
+    %p
+      Set the <code>allow_multiple_attempts</code> property to <code>false</code> in the DSL
+      to give students only a single opportunity to answer the question. By default, students
+      are allowed multiple attempts.
+
+  - if @level.is_a?(External)
+    %h4 Tip: Bubble Choice Options
+    %p
+      If this level will be a sublevel of a BubbleChoice level, you can set the following properties in the DSL:
```

Comment:
I wonder if it's worth adding an equivalent tip to bubble choice levels under the component you built to edit these settings in the parent level - "if any of these sublevels are markdown levels, you cannot use the settings on this page to set the title, description, and thumbnail. They need to be set in the DSL settings of the actual level". Not blocking to this work; just potential follow-up wonder.